tests/conftest.py

```diff
@@ -966,3 +966,12 @@ def system_manager_client(db, system):
     db.refresh(client)
     yield client
     client.delete(db)
+
+
+@pytest.fixture(scope="function")
+def load_default_data_uses(db):
```

Comment:
i didn't dig into this too much, but i needed to put this in to stop our scaffolding from holding me up in my testing. if this feels too hacky, i can dig into this further now that other pieces are in place